examples/agents/biochemist.py

```diff
@@ -18,18 +17,14 @@
 llm = None
 summary_model = None
 if "claude" in model:
-    import anthropic
-    log10(anthropic)
```

Comment:
Didn't quite understand why this and the other examples need to be updated. We aren't deprecating the wrapper approach correct?

Reply:
This was in the spirit of having a non-openai/anthropic way of tracing calls. Is unrelated to the logger, so we can chop that off

Reply:
(rebased this of the previous PR)

Reply:
It actually wouldn't work together - you'd get double instrumentation

Reply:
Removed for now